### PR TITLE
Add handlebars style guide with simple rules

### PR DESCRIPTION
**Pull request by drspaniel**

### CODE REVIEW

File: engineering/handlebars.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Table Of Contents
+
+* [Quotes](#quotes)
+* [Multiline expressions](#multiline-expressions)
+
+## Quotes
+Prefer double quotes to single quotes
+```hbs
+<!-- good -->
```
Comment (brzpegasus) [inline engineering/handlebars.md]: Probably best to use handlebars comments: `{{! good }}`

Comment (drspaniel) [inline engineering/handlebars.md]: I preferred html comments for some reason, but a quick poll in #fungineering on slack showed people preferring hbs comments. Will change.

Comment (brzpegasus) [inline engineering/handlebars.md]: In this context, it doesn't really matter, but for templates in general, hbs comments won't show up in the resulting markup, whereas html comments will.

**Comment by duggiefresh** [comment]
I like this and it's becoming commonplace already in the current projects. :+1:

**Comment by drspaniel** [comment]
Well @poteto very kindly requests on every one of my PRs so :p

**Comment by poteto** [comment]
Haha yes. I like this! :+1:

**Comment by duggiefresh** [comment]
BOOM!

**Pull request merged**